aurora/bag_transfer/lib/transfer_routine.py

```diff
@@ -103,37 +90,24 @@ def run_routine(self):
 
         return self.transfers if self.transfers else False
 
-    #####################
-    # START SETUP METHODS
-    #####################
     def has_active_organizations(self):
-        self._setup_active_organizations()
-        return True if self.active_organizations else False
-
-    def _setup_active_organizations(self):
+        """Checks to see if the routine has active organizations."""
         self.active_organizations = Organization.objects.filter(is_active=True)
+        return True if self.active_organizations else False
 
     def verify_organizations_paths(self):
-        ck = 0
+        """Makes sure organizations have upload and processing paths."""
         orgs_to_remove = []
         for org in self.active_organizations:
             if not all_paths_exist(org.org_machine_upload_paths()):
-                ck += 1
                 orgs_to_remove.append(org)
-            # TODO: PLACE ERR
-        # remove org from Routine and log
-        if ck > 0:
-            self.active_organizations = [
-                org for org in self.active_organizations if org not in orgs_to_remove
-            ]
-            Pter.plines(
-                ["{} orgs were removed from routine".format(len(orgs_to_remove))]
-            )
+        self.active_organizations = [org for org in self.active_organizations if org not in orgs_to_remove]
 
     def build_contents_dictionary(self):
+        """Creates a dictionary of bags to process."""
         org_dir_contents = {}
         for org in self.active_organizations:
-            upload_dir = org.org_machine_upload_paths()[0]
+            upload_dir, _ = org.org_machine_upload_paths()
```

Comment:
Curious about why you changed the approach here--

Reply:
Eh, it's a preference thing but I find this a bit more readable than slicing.